chunks/lang-java/99-gradle.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+
+# Extract the description field from the JSON string
+description=$(echo "$GITPOD_WORKSPACE_CLASS_INFO" | jq -r '.description')
+
+# Extract the number of vCPUs from the description field
+# Assumes the format "8 vCPU, 16GB memory, 50GB disk"
+vcpus=$(echo "$description" | grep -oP '\d+ vCPU' | awk '{print $1}')
+
+# we leave 2 vCPUs for other processes, such as IDE backends
+workers=$((vcpus - 2))
```

Comment:
What about workspaces with 2 vCPUs? Maybe 
```suggestion
workers=$((vcpus - 2))
workers=$(( workers < 1 ? 1 : workers ))
```